app/src/main/java/emu/skyline/MainActivity.kt

```diff
@@ -304,14 +294,18 @@ class MainActivity : AppCompatActivity() {
     }
 
     private fun populateAdapter() {
+        binding.textTitle.text = null
+        binding.textTitle.visibility = View.GONE
         val items = getDataItems()
         adapter.setItems(items.map {
             when (it) {
```

Comment:
Couldn't you drop most of this? The whole reason for the map and when was that there could be more than one type. That's no longer the case.